client/packages/system/src/Stock/ListView/AppBarButtons.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import {
+  DownloadIcon,
+  useNotification,
+  AppBarButtonsPortal,
+  Grid,
+  useTranslation,
+  FileUtils,
+  LoadingButton,
+  SortBy,
+  EnvUtils,
+  Platform,
+} from '@openmsupply-client/common';
+import { StockLineRowFragment, useStock } from '../api';
+import { stockLinesToCsv } from '../../utils';
+
+export const AppBarButtonsComponent: FC<{
+  sortBy: SortBy<StockLineRowFragment>;
+}> = ({ sortBy }) => {
+  const { success, error } = useNotification();
+  const t = useTranslation(['distribution', 'common']);
+  const { fetchAsync, isLoading } = useStock.line.listAll(sortBy);
+
+  const csvExport = async () => {
+    const data = await fetchAsync();
+    if (!data || !data?.nodes.length) {
+      error(t('error.no-data'))();
+      return;
+    }
+
+    const csv = stockLinesToCsv(data.nodes, t);
+    FileUtils.exportCSV(csv, t('filename.outbounds'));
+    success(t('success'))();
+  };
+
+  return (
+    <AppBarButtonsPortal>
+      <Grid container gap={1}>
+        <LoadingButton
+          startIcon={<DownloadIcon />}
+          isLoading={isLoading}
+          variant="outlined"
+          onClick={csvExport}
+          disabled={EnvUtils.platform === Platform.Android}
```

Comment:
So it's disabled on Android? Might it be better to just hide it completely in that case?

Reply:
For UI items, the preference is for functionality which isn't available to still be shown - it tells the user that they can do something.. as long as they have the right platform / permissions etc. I prefer having an uncluttered UI, but can see the point.

The android fix was a quick and dirty one - with a plan to revisit and implement later. I'd prefer to keep the button visible, as a reminder to make it work sometime.